code/src/Form.js

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react';
+
+import { NameQuestion } from './components/NameQuestion'
+import { DropDown } from './components/DropDown'
+import { RadioButton } from './components/RadioButton'
+import { Summary } from './components/Summary'
+
+export const Form = () => {
+    const [name, setName] = useState('');
+    const [brand, setBrand] = useState('');
+    const [knowHeadPhone, setKnowHeadPhone] = useState('');
+    const [summary, setSummary] = useState(false);
+
+    const handleSubmit = event => {
+        event.preventDefault();
+        setSummary(true)
+    };
+
+    return (
+        <>
+            {!summary ? (
+                <form onSubmit={handleSubmit} className="form-container" >
+                    <NameQuestion name={name} setName={setName} />
+                    <DropDown brand={brand} setBrand={setBrand} />
+                    <RadioButton knowHeadPhone={knowHeadPhone} setKnowHeadPhone={setKnowHeadPhone} />
```

Comment:
I like the way you use props. 👍 

You have always completed the conditional rendering which is great! If you have the energy, you can either for this week or a future week have a look at the && and create a multi-step page